addons/web_editor/tools.py

```diff
@@ -97,6 +101,13 @@ def get_video_url_data(video_url, autoplay=False, loop=False, hide_controls=Fals
             params['controls'] = 0
         if loop:
             params['loop'] = 1
+        groups = platform_match.groupdict()
+        if 'hash' in groups:
+            params['h'] = groups['hash']
+        elif 'params' in groups:
```

Comment:
Isn't it possible to have a hash AND some params?

Reply:
It's possible that people provide additional parameters but they will be ignored. We actually build a new url based on the parameters the user set on the configuration modal (on the modal we open when the user uses the `/video` command). We will now only retrieve the video id and the hash parameter from the url the user provide.